src/Pact/Analyze/Model/Text.hs

```diff
@@ -185,11 +196,17 @@ showEvent ksProvs tags event = do
             displayCallScope :: Text -> Pact.ModuleName -> Text -> [Text]
             displayCallScope noun modName funName =
               -- TODO(joel): convert all of this to Doc
-              let header = renderCompactText' $
-                           "entering " <> pretty noun <> " " <> pretty modName
-                        <> "." <> pretty funName <> " with "
-                        <> if length vids > 1 then "arguments" else "argument"
-              in header : (displayVids showArg ++ [emptyLine])
+              let withArgs = case length vids of
```

Comment:
Aside: I've been thinking it would be nice to use `Doc` instead of `Text` in this module